HawkesRLTrading/src/SimulationEntities/TradingAgent.py

```diff
@@ -292,4 +292,70 @@ def getobservations(self):
         rtn={"Cash":self.cash,
              "Inventory": self.Inventory[self.exchange.symbol],
              "Positions": self.positions[self.exchange.symbol]}
-        return rtn
\ No newline at end of file
+        return rtn
+    
+class TradingAgent(TradingAgent):
```

Comment:
why are you redefining the TradingAgent class? why not edit the parent class itself?